src/overlays/actors/ovl_En_Changer/z_en_changer.c

```diff
@@ -216,31 +216,31 @@ void EnChanger_OpenChests(EnChanger* this, PlayState* play) {
     f32 zPos;
     f32 yPos;
     f32 xPos;
-    s16 temp_s0;
-    s16 temp_s0_2;
+    s16 openedChestSide;
+    s16 unopenedChestItemType;
     EnBox* left;
     EnBox* right;
 
     left = this->leftChest;
     right = this->rightChest;
 
     if (this->timer == 0) {
-        temp_s0_2 = temp_s0 = this->chestOpened; // Required to use the right registers
+        unopenedChestItemType = openedChestSide = this->chestOpened; // Required to use the right registers
```

Comment:
This isnt necessarily related to your changes, but `this->chestOpened` reads to me as a boolean "is chest opened". Didnt realize it was actually the side.
Do you think it makes sense to call it `openedChestSide` like the name you gave to the temp?

(also, feel free to remove that comment about it being required to match the asm registers)

Reply:
`selectedChest` might be a nicer name for it

Reply:
oh yeah, i agree

Reply:
beyond the name though, Im still wondering if the temp and the instance variable should have the same name. I think the answer is yes? so `selectedChest` for both

Reply:
The register comment seems useful to me since it doesn't make sense why `unopenedChestItemType` would be set at that point only to be reassigned later. I'm good with it either way though.

Reply:
I dont feel like its necessary to comment on asm specifics, at least to this degree.
maybe 
```
// `unopenedChestItemType` being set here is required for matching, but it doesnt do anything useful
unopenedChestItemType = openedChestSide = this->chestOpened;
```